bigquery/tables.js

```diff
@@ -104,6 +104,37 @@ function deleteTable (options, callback) {
 }
 // [END delete_table]
 
+// [START copy_table]
+/**
+ * Create a copy of an existing table
+ *
+ * @param {object} options Configuration options.
+ * @param {string} options.srcDataset The source dataset ID.
+ * @param {string} options.srcTable The source table ID.
+ * @param {string} options.destDataset The destination dataset ID.
+ * @param {string} options.destTable The destination table ID. Will be created if it doesn't exist.
+ * @param {function} callback The callback function.
+ */
+function copyTable (options, callback) {
```

Comment:
Instead of a single `options` argument, switch to four positional arguments, e.g. 

```js
function copyTable (srcDataset, srcTable, destDataset, destTable, callback) {
```

Plus all necessary changes that will need to be made to the CLI/tests.